rundk/main.go

```diff
@@ -93,34 +101,42 @@ func setup() (interactive.Docker, func()) {
 				cleanup := cmd.CopyAndAddMount("bind", tmpDir, gcloudKey, gcloudKey)
 				builtUpDefers = append(builtUpDefers, cleanup)
```

Comment:
Don't you need to set `envs["CLOUDSDK_CONFIG"]` in this branch of the if-else statement?

Reply:
To add a bit more contexts here:
- `GOOGLE_APPLICATION_CREDENTIALS` is for authenticating with a Google service account
- The credential files under the gcloud config folder are for authenticating with a user account

So we don't need to set the `CLOUDSDK_CONFIG` env var if we are using the `GOOGLE_APPLICATION_CREDENTIALS` env var, since it's only used for overwriting the default value for gcloud config path, which is for a different authentication mechanism.